src/dwalk/dwalk.c

```diff
@@ -303,7 +303,7 @@ static void print_usage(void)
     printf("Options:\n");
     printf("  -i, --input <file>      - read list from file\n");
     printf("  -o, --output <file>     - write processed list to file in binary format\n");
-    printf("  -t, --text              - use with -o; write processed list to file in ascii format\n");
+    printf("  --text-output <file>    - write processed list to file in ascii format\n");
```

Comment:
Let's rename this option to ``--output-text`` just to keep it close to ``--output`` if we decide to list these alphabetically.